src/components/Alert/AlertLink.tsx

```diff
@@ -12,7 +12,11 @@ export const AlertLink = ({
   isExternal
 }: AlertLinkProperties): JSX.Element => (
   <li className='m-list_item'>
-    <a className='m-list_link' href={href}>
+    <a
+      className='m-list_link'
+      href={href}
+      aria-label={`${label || ''} external link}`}
```

Comment:
NIT: I'd probably add an `aria-label` to the external link icon rather than the anchor, that way you can rely on the normal label text.